benchmarks/tailwind-component/src/benchmark-0.0.12-vs-local-version.ts

```diff
@@ -1,18 +1,21 @@
 import { render } from "@react-email/render";
+import { writeFile } from "fs/promises";
```

Comment:
could we use the sync version for `writeFile`? since we are _awaiting_ for it anyways and we also wouldn't have to turn the `.then()` callback into an async function